doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst

```diff
@@ -0,0 +1,154 @@
+.. _ncs_release_notes_2699_cs2:
+
+|NCS| v2.6.99-cs2 Release Notes
+###############################
+
+.. contents::
+   :local:
+   :depth: 3
+
+|NCS| v2.6.99-cs2 is a **customer sampling** release, tailored exclusively for participants in the nRF54H20 customer sampling program.
+Do not use this release of |NCS| if you are not participating in the program.
+
+The release is not part of the regular release cycle and is specifically designed to support early adopters working with the nRF54H20 device.
+It is intended to provide early access to the latest developments for nRF54H20, enabling participants to experiment with new features and provide feedback.
+
+The scope of this release is intentionally narrow, concentrating solely on the nRF54H20 device.
+You can use the latest stable release of |NCS| to work with other boards.
+
+All functionality related to nRF54H20 is considered experimental.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Peripherals support
+===================
+
+* Added experimental support for the following peripherals on the nRF54H20 DK:
+
+  * ADC
+  * CLOCK
+  * GPIO
+  * GRTC
+  * I2C
+  * MRAM
+  * PWM
+  * RADIO (Bluetooth Low Energy and 802.15.4)
+  * SPI
+  * UART
+  * USB
+  * WDT
+  * NFCT
+
+Working with nRF54H Series
+==========================
+
+* Added:
+
+  * The :ref:`ug_nrf54h20_gs` page.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Enhanced ShockBurst (ESB)
+-------------------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` board.
+* Added fast switching between radio states for the nRF54H20 SoC.
+* Added fast radio channel switching for the nRF54H20 SoC.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+Bluetooth samples
+-----------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpuapp`` in the following Bluetooth samples:
```

Comment:
The following samples support the nrf54h20dk_nrf54h20_cpuapp target:
- central_and_peripheral_hr
- central_bas
- central_hids
- central_hr_coded
- central_uart
- multiple_adv_sets
- peripheral_ams_client
- peripheral_ancs_client
- peripheral_bms
- peripheral_cgm
- peripheral_cts_client
- peripheral_gatt_dm
- peripheral_hids_keyboard
- peripheral_hids_mouse
- peripheral_hr_coded
- peripheral_lbs
- peripheral_nfc_pairing
- peripheral_rscs
- peripheral_status
- peripheral_uart
- shell_bt_nus
- throughput

Reply:
I don't think these are in scope of this release